GameData/RealismOverhaul/Engine_Configs/NK33_Config.cfg

```diff
@@ -298,11 +301,11 @@
 	TESTFLIGHT
 	{
 		name = NK-33
-		ratedBurnTime = 240
+		ratedBurnTime = 365 // [10]
```

Comment:
All uses of this engine I could find burned from roughly 113 seconds (N1 1964) to 139 seconds (Kistler K-1), what is the 365 from?

IIRC, Agathorn's standard is to add ~5% to the longest burn time you find to give it a little leeway?

Same issue exists with the ratedBurnTime on other versions.

Ahh, you cite source [10]. Can you help me find a link to the full source? It appears to be from 1991, which may not be the best source to lean on when we unlock it in the 60s, but I understand we take what we can get for sources. I'm definitely willing to read the source and give 365s a fair shot :)